app/domain/healthchecks/monthly_aggregations.rb

```diff
@@ -0,0 +1,23 @@
+module Healthchecks
+  class MonthlyAggregations
+    include Concerns::Deactivable
+
+    def name
+      :aggregations
+    end
+
+    def status
+      aggregations.positive? ? :ok : :critical
+    end
+
+    def message
+      "ETL :: no aggregations of metrics for yesterday" if status == :critical
+    end
+
+  private
+
+    def aggregations
+      @aggregations ||= Aggregations::MonthlyMetric.where(created_at: Date.yesterday).count
```

Comment:
Still, although the ETL process runs daily and retrieves the metrics for the day before, it seems to me that the `created_at` attribute should be `today`?

Could you please add an index for this field as well?

Reply:
Thanks - addressed in https://github.com/alphagov/content-performance-manager/pull/1153/commits/01a92df30f727fd33857bba12d4ada76cd6471ae and added index in https://github.com/alphagov/content-performance-manager/pull/1153/commits/5adefb394a85ae7c3a6fd9872cd16c4330695525